lighthouse-core/test/chromium-web-tests/run-web-tests.sh

```diff
@@ -6,81 +6,13 @@
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 ##
 
-set -euo pipefail
```

Comment:
`set -u` at least still seems relevant, no?

Reply:
@patrickhulce Adding `set -u` causes the DevTools check to fail in GitHub check.  Seems to be failing on the `yarn --frozen-lockfile` step: https://github.com/GoogleChrome/lighthouse/runs/1327351848.

This appears to be a failure of the script but I can't tell where. What do you make of this?

Reply:
Hmm maybe `set -u` was not the problem?

Reply:
Yeah that error has been happening for the past several weeks (maybe even months?) and seems like we hit some sort of github network troubles after downloading all of chromium build tooling all the time heh 😄 

I think this is unrelated to `set -u`